change release engineer to base load engineer (#4917)
mozilla/fx-private-relay: docs/base-load-engineer-playbook.md

# Relay: "Base load" Engineer playbook

Starting in H2'24, Relay ENGR team expanded the "Release Engineer" role into a "Base
load Engineer" role who handles consistent expected work like releases and dependency
updates, and who fields un-planned incoming work requests like bug reports and customer

> support requests. (The term "base load" comes from electrical grids, where it describes
> the minimum level of demand over a span of time. https://w.wiki/Anad)

Reviewer: Nit: why use a URL shortner here?

```suggestion
support requests. (The term "[base load][]" comes from electrical grids, where it describes
the minimum level of demand over a span of time.)

[base load]: https://en.wikipedia.org/wiki/Base_load
```

You don't have to perform every task yourself. When you check any of the channels below,
you may delegate the resulting task to the most appropriate party.

1. Check [Security Dependabot Alerts][security-dependabot-alerts] for any critical
   security updates to make
2. Check #relay-alerts for any critical operational issues to fix
3. Check the [Sentry Releases][sentry-releases] report to watch for any new issues
4. Check #relay-jira-triage for any urgent new tickets
5. Check #privsec-customer-experience channel for any urgent inbound CX requests
6. Check [dependabot pull requests][dependabot-prs].
   - See the [Dependency Updates doc][dependency-updates-doc].
7. Co-ordinate, re-tag for [stage fixes][stage-fixes] as needed

## Mondays

1. Daily routine
2. Prepare release for tomorrow
   - Look thru [What's Deployed][whats-deployed] tool to make sure all the
     commits on stage are ready to go to prod.
   - You can also run a [comparison on GitHub][github-compare] between [the stage
     version][stage-version] and [the prod version][prod-version].
   - [Write SRE ticket, mention authors, reference in release notes][release-to-prod]

## Tuesdays

> 1. Dialy routine

Reviewer 2: ```suggestion
1. Daily routine
```

2. SRE processes ticket to release the tag to production
3. Update [Github Release][github-releases] to current release

> 4. Monitor [Sentry Releases] for new production issues

Reviewer 2: ```suggestion
4. Monitor [sentry-releases] for new production issues
```
